ecs_fargate/README.md

```diff
@@ -35,67 +35,95 @@ To monitor your ECS Fargate tasks with Datadog, run the Agent as a container in
-13. Add another environment variable using the **Key** `ECS_FARGATE` and the value `true`. Click **Add** to add the container.
-14. Add another environment variable using the **Key** `DD_SITE` and the value {{< region-param key="dd_site" code="true" >}}. This defaults to `datadoghq.com` if you don't set it.
-15. (Windows Only) Select `C:\` as the working directory.
-16. Add your other containers such as your app. For details on collecting integration metrics, see [Integration Setup for ECS Fargate][8].
-17. Click **Create** to create the task definition.
-
-##### AWS CLI
-
-1. Download [datadog-agent-ecs-fargate][9]. **Note**: If you are using Internet Explorer, this may download as gzip file, which contains the JSON file mentioned below.**
-2. Update the JSON with a `TASK_NAME`, your [Datadog API Key][6], and the appropriate `DD_SITE` ({{< region-param key="dd_site" code="true" >}}). **Note**: The environment variable `ECS_FARGATE` is already set to `"true"`.
-3. Add your other containers such as your app. For details on collecting integration metrics, see [Integration Setup for ECS Fargate][8].
+The instructions below show you how to configure the task using the [Amazon Web Console][4], [AWS CLI tools][5], or [AWS CloudFormation][6].
+
+{{< tabs >}}
+{{% tab "Web UI" %}}
+##### Web UI Task Definition
+
+1. Log in to your [AWS Web Console][1] and navigate to the ECS section.
+2. Click on **Task Definitions** in the left menu, then click the **Create new Task Definition** button or choose an existing Fargate task definition.
+3. For new task definitions:
+    1. Select **Fargate** as the launch type, then click the **Next step** button.
+    2. Enter a **Task Definition Name**, such as `my-app-and-datadog`.
+    3. Select a task execution IAM role. See permission requirements in the [Create or Modify your IAM Policy](#create-or-modify-your-iam-policy) section below.
+    4. Choose **Task memory** and **Task CPU** based on your needs.
+4. Click the **Add container** button to begin adding the Datadog Agent container.
+    1. For **Container name** enter `datadog-agent`.
+    2. For **Image** enter `public.ecr.aws/datadog/agent:latest`.
+    3. For **Env Variables**, add the **Key** `DD_API_KEY` and enter your [Datadog API Key][2] as the value.
+    4. Add another environment variable using the **Key** `ECS_FARGATE` and the value `true`. Click **Add** to add the container.
+    5. Add another environment variable using the **Key** `DD_SITE` and the value {{< region-param key="dd_site" code="true" >}}. This defaults to `datadoghq.com` if you don't set it.
+    6. (Windows Only) Select `C:\` as the working directory.
+11. Add your other application containers to the task definition. For details on collecting integration metrics, see [Integration Setup for ECS Fargate][3].
```

Comment:
nit: why is this 11 ? (same below with 12)

Reply:
Ah just missed that when cleaning those up. Hugo doesn't care about what actual numbers are used here and will figure out what numbers to use in the bullet points itself. So the resulting page was clean, but will clean those up.